pubpub/pubpub
# code block syntax highlighting language selection

Implements language selection dropdown in code blocks.

![Screenshot from 2022-09-26 10-04-09](https://user-images.githubusercontent.com/14115194/192297001-ca72c682-e383-42f3-b6b8-9a4f7f927ccb.png)


_test plan_
1. create new pub
2. create code block either by clicking the code block icon in the toolbar (looks like a code lego) or  by key combination ctrl + shift + \
3. type some JS into code block
4. click on code block, dropdown should appear
5. set language to JS
6. syntax highlighting for JS should appear
7. set language to something else by same approach
8. js highlighting should be gone
9. refresh -> language selection should persist
